docs-source/content/userguide/introduction/architecture.md

```diff
@@ -67,7 +67,7 @@ During a rolling event caused by a change to the Domain's `image` field, contain
 Within the container, the following aspects are configured by the operator:
 
 *	The `ENTRYPOINT` is configured by a script that starts up a Node Manager process, and then uses WLST to request that Node Manager start the server.  Node Manager is used to start servers so that the socket connection to the server will be available to obtain server status even when the server is unresponsive.  This is used by the liveness probe.
-* The liveness probe is configured to check that the server is alive by querying the Node Manager process.  The liveness probe is by default configured to check liveness every 15 seconds, and to timeout after 5 seconds.  If a pod fails the liveness probe, Kubernetes will restart that container.
+* The liveness probe is configured to check that the server is alive by querying the Node Manager process.  The liveness probe is by default configured to check liveness every 15 seconds, and to timeout after 5 seconds.  If a pod fails the liveness probe, Kubernetes will restart that container. It is possible to customize the liveness probe by using `livenessProbeCustomScript` field in the domain resource or by setting `LIVENESS_PROBE_CUSTOM_SCRIPT` environment variable. These options can be used to provide a custom script which will be invoked by the liveness probe script (`livenessProbe.sh`) for WebLogic server Pods. If this custom script fails with non-zero exit status, the liveness probe will fail and Kubernetes will restart the container.
```

Comment:
1) maybe add a mention that the liveness probe interval and timeout can be customized in the domain resource

2) make it clear that a custom liveness probe is not a replacement for the current probe -- it's in addition to the current probe 

e.g. something like:

will be invoked by the liveness probe script (`livenessProbe.sh`) for WebLogic server Pods -->

will be invoked by the existing liveness probe script (`livenessProbe.sh`) for WebLogic server Pods after the existing script performs its own checks

Reply:
It seems the liveness probe interval and timeout can be tuned using tuning parameters in operator's config map. I didn't see an option to customize it in domain resource. We don't document all the tuning parameters and am not sure if customers have frequent need to tune these liveness probe tuning parameters or not.

Reply:
@rjeberhard thoughts?

Reply:
It looks like I can specify this in serverPod section of domain resource. I'll mention that in the documentation.